### DISCUSSION TITLE
part of the release version being an upstream version

### DISCUSSION THREAD

**Discussion by sylvainOL** [semantic-release/semantic-release#2480] [Q&A]
Hello,
I tried to find an answer for that but didn't found.

We are using an upstream component and main part of our repo is to create few files around.
therefore, I'd like to use tag version of the upstream component (which is available in a file) and add a `build` tag starting with zero.

here's a quick example:

* upstream is `v1.0.0` and we don't have any version on this one --> `v1.0.0-r0`
* upstream is still `v1.0.0` and we make a change in the build files --> `v1.0.0-r1`
* upstream moves to `v1.1.0` --> `v1.1.0-r0`

this is common pattern for "packaging" components :
* [debian "nano" package on bookworm](https://packages.debian.org/fr/source/bookworm/nano) is versionned as `6.3-1` where `6.3` is the last version of nano
* [rke2, a kubernetes installer](https://github.com/rancher/rke2) last version is `v1.24.1+rke2r2` and previous one was `v1.24.1+rke2r1`

Is it possible to achieve such pattern with semantic-release?

thanks

**Comment by travi** [comment]
A core purpose of semantic-release is to encourage proper use of semantic versioning in order to communicate the impact of the changes in each release. While basing a version on outside information may be a common enough practice, it does not communicate the impact of the changes in that release of that package. If your goal is to use outside information, semantic-release is not the tool you are looking for.
